examples_tests/50.MeshPacking/main.cpp

```diff
@@ -0,0 +1,190 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.

```

Comment:
actually integrate this into example 41 Visibility Buffer

Reply:
so what's happening now. I see both an ex 50 and and a renewed 41?

Reply:
I will delete ex 50 later, now i need it for testing purpose